### PR TITLE
Morse decode

### PR DESCRIPTION
**Pull request by marilirulita**
### In this project, we run a program built in Ruby to decrypt messages written in Morse code. This branch creates a few methods to automate this process and reuse these whenever possible.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  word = ''
+  word_array.each { |c| word << decode_morse_code_character(c) }
+  word
+end
+
+def decode_morse_code_message(incoded_message)
+  message_array = incoded_message.split('   ')
+  message = ''
+  message_array.each { |word| message << " #{decode_morse_code_word(word)}" }
+  message
```
Comment (DammyShittu) [inline main.rb]: - [ ] From the image below, it appears there is a leading space in the decoded sentences. If you are struggling with white space simply add `.strip` to the `message` in line 26 to fix it. i.e. `message.strip`

  Result | Expected result
  :---:|:---:
  trim | trim-main

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi team,

✔️ Decoding of word and character

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
@DammyShittu Thank you for the review. we will make the changes now.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @marilirulita, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You did well to make the changes requested by the previous reviewer 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TadesseDev** [comment]
Thank you @elisha2kyakpo1 @DammyShittu. Great to have your reviews and confirmation.

**Pull request merged**